ibtk/include/ibtk/PoissonFACPreconditioner.h

```diff
@@ -94,6 +94,11 @@ class PoissonFACPreconditioner : public FACPreconditioner, public PoissonSolver
     void setPoissonSpecifications(const SAMRAI::solv::PoissonSpecifications& poisson_spec);
 
     /*!
+     * \brief Set the scale factor for C and D in the SAMRAI::solv::PoissonSpecifications object.
+     */
+    void setCoefScales(double C_scale, double D_scale);
+
```

Comment:
Shouldn't this be set through the PoissonSpecifications? We don't want redundant data specification.

Reply:
I also thought about this. But I think this is much safer and less memory consuming. One can multiply and then divide (scale-revert back) SC/NC coefs variables, but this is prone to double precision error accumulation. Other way is to create temporary scaled coefs, but this uses extra memory. This was a compromise and a design decision. Although I agree that  using PoissonSpecifications is much more elegant, and was my first choice also. I also thought about adding these scales in PossionSpec object directly, but that would be kind of same thing, just a different flavor.